packages/core/src/format/format.ts

```diff
@@ -177,7 +168,7 @@ export function formatProblems(
         fingerprint: `${p.ruleId}${p.location.length > 0 ? '-' + p.location[0].pointer : ''}`,
       };
     });
-    process.stdout.write(JSON.stringify(issues, null, 2));
+    logger.info(JSON.stringify(issues, null, 2));
```

Comment:
I believe this doesn't make sense in the browser environment. We do both `info` and `error` into `process.stderr`. As far as I remember we only use `stdout` to convey some data in Workflows.

Reply:
but `error` and `warn` write to `stderr`

Reply:
I mean, we keep `stdout` only for some technical purposes, and put any user messages to `stderr`.

Reply:
should I create helper like `output` with one method `write` ? Which would write to logger in browser env ?

Reply:
That makes sense.